queue_job/jobrunner/__init__.py

```diff
@@ -66,9 +66,13 @@ def prefork_start(server, *args, **kwargs):
     global runner_thread
     res = orig_prefork_start(server, *args, **kwargs)
     if not config['stop_after_init']:
-        _logger.info("starting jobrunner thread (in prefork server)")
-        runner_thread = QueueJobRunnerThread()
-        runner_thread.start()
+        if (_channels().strip().startswith('root:0')):
+            _logger.info("jobrunner thread (in prefork server) NOT started, " \
+                         "because the root channel's capacity is set to 0")
```

Comment:
It would be nice to factor out this part in a utility method (runner_enabled or something).

Otherwise looks good to me.